.github/workflows/ci.yaml

```diff
@@ -64,6 +64,26 @@ jobs:
           version: v1.61
           args: --timeout=10m
 
+  security-scan:
+    name: Security Scan
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/golang@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Hey @chandankumar4 - were you able to create this secret? I see it in the list of Secrets as something that was created an hour ago.

Reply:
yes, I created the secret